.htaccess

```diff
@@ -49,7 +49,27 @@ deny from env=stayout
 
     # forbidden folders
     RewriteRule .*\.php - [F]
-    RewriteRule .*\.twig - [F]
+    RewriteRule \.php5 - [F]
```

Comment:
Q: Any reason we use `.*\.php` and the other rules don't have the wildcard?

Reply:
I think it had to do with the index.php but I'll have a look if we can tighten it more